mobileiron_api/api/base_api.py

```diff
@@ -50,9 +52,9 @@ def _execute_call(self,
                       params: str = None,
                       json_value: object = None):
         response = None
+        if params is not None:
+            url = f"{url}?{params}"
         if http_method == 'get':
```

Comment:
Does this block serve a purpose

Reply:
it was included only in 'get' method, so I put it before going through the method selection